packages/www/components/Register/index.tsx

```diff
@@ -123,6 +124,20 @@ const Register = ({ id, buttonText, onSubmit, loading, errors }) => {
             onChange={(e) => setFirstName(e.target.value)}
           />
 
+          <TextField
```

Comment:
Does this already go magically on the request sent to backend? I think it's called `lastName` on the API https://github.com/livepeer/livepeer-com/blob/4e24f8e656e204089b0d0057428820045c162c89/packages/api/src/controllers/user.ts#L314

Reply:
@victorges good call, you were right - I updated it (it's `lastName` in Hubspot too) and tested locally the the last name makes it into the DB _and_ Hubspot